pkg/ottl/ottlfuncs/README.md

```diff
@@ -444,11 +246,209 @@ Examples:
 
 - `truncate_all(resource.attributes, 50)`
 
+## Converters
+
+Converters are functions that help translate between the OTTL grammar and the underlying pdata structure.
+They manipulate the OTTL grammar value into a form that will make working with the telemetry easier or more efficient.
```

Comment:
Can you add a sentence in here that Converters never change the underlying telemetry.

Reply:
We did have one right below this paragraph, but each section was essentially reiterating the same information, so I consolidated and reworded them.

Reply:
I'm blind 👍